about.html

```diff
@@ -2,406 +2,109 @@
-							development and its context.
-						</p>
-					</div>
-					<div class="program">
-						<span
-							><svg
-								version="1.1"
-								id="Capa_1"
-								xmlns="http://www.w3.org/2000/svg"
-								xmlns:xlink="http://www.w3.org/1999/xlink"
-								x="0px"
-								y="0px"
-								width="50px"
-								height="50px"
-								viewBox="0 0 50 50"
-								style="enable-background: new 0 0 50 50"
-								xml:space="preserve"
-							>
-								<g>
-									<path
-										fill="white"
-										d="M37.448,20.62l3.557-4.826v-1.521h-2.667v-4.44c0,0-0.691-0.566-3.302-1.06l-0.72-2.402
-                               c-0.014-0.043-0.039-0.081-0.075-0.11l-6.179-4.84c0.043-0.117,0.074-0.238,0.074-0.368c0-0.58-0.471-1.053-1.053-1.053
-                               S26.03,0.473,26.03,1.053c0,0.581,0.472,1.054,1.053,1.054c0.289,0,0.553-0.118,0.742-0.308l6.086,4.769l0.636,2.121
-                               C32.521,8.345,29.5,8.055,25,8.055c-11.174,0-13.337,1.778-13.337,1.778v4.44H8.996v1.521l3.557,4.826h0.887l1.751,24.373
-                               l-3.528,3.819V50h26.674v-1.188l-3.528-3.819L36.56,20.62H37.448z"
-									/>
-								</g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g></svg
-						></span>
-						<h3>Lecture</h3>
-						<p>
-							Listen to the speakers from various countries about the message of
-							development and its context.
-						</p>
-					</div>
-					<div class="program">
-						<span
-							><svg
-								version="1.1"
-								id="Layer_1"
-								xmlns="http://www.w3.org/2000/svg"
-								xmlns:xlink="http://www.w3.org/1999/xlink"
-								width="50px"
-								height="50px"
-								viewBox="0 0 512 512"
-								style="enable-background: new 0 0 512 512"
-								xml:space="preserve"
-								fill="white"
-							>
-								<g>
-									<g>
-										<polygon
-											points="302.46,46.46 302.46,0 209.54,0 209.54,46.46 8.213,46.46 8.213,92.92 54.673,92.92 54.673,371.681 
-                                   207.661,371.681 100.194,479.15 133.046,512 256,389.046 378.955,512 411.807,479.15 304.339,371.681 457.327,371.681 
-                                   457.327,92.92 503.787,92.92 503.787,46.46 		"
-										/>
-									</g>
-								</g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g>
-								<g></g></svg
-						></span>
-						<h3>Exhibition</h3>
+				<span class="site-map">Home <span class="gt">&gt;</span> About</span>
+				<section class="show-case">
+					<h1 class="">African Development Forum Global Summit 2022</h1>
+					<div class="detail">
```

Comment:
- [ ] Please correct the size of this section on the about page to be similar to the original version 👇🏽 

| Original version | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/197134803-ab237323-5e53-4d03-a9a6-63128eb7fab9.png) | ![image](https://user-images.githubusercontent.com/58642949/197134917-07f25f17-a01a-4316-a0e7-681178424aa9.png) |